src/teehr/loading/nwm/retrospective_grids.py

```diff
@@ -299,20 +302,50 @@ def nwm_retro_grids_to_parquet(
         weight_vals = weights_df.weight.values
 
         if chunk_by is None:
-            gps = [(None, var_da)]
```

Comment:
Lets see if we can make the nwm21 and nwm30 more similar and maybe add a few tests.